webapp/src/main/webapp/src/app/receiver/receiver.component.html

```diff
@@ -7,7 +7,7 @@
 			</ng-template>
 		</li>
 		<li ngbNavItem="syncMsgs">
-			<a ngbNavLink i18n="@@common-sync-msgs">Sync Items</a>
+			<a ngbNavLink i18n="@@receiver-pending-events">Sync Items</a>
```

Comment:
They are actually **Sync Items**, FYI the label here says **Sync Items** but in the translation file it **Pending Events**